app/javascript/api/listingsApiService.ts

```diff
@@ -56,6 +56,9 @@ export const getListings = async (
         ({ data }) => data.listings
       )
 
+export const getListing = async (listingId?: string): Promise<ListingsResponse> =>
```

Comment:
my guess is that we'll want a separate listingApiService file because we'll be making a few other listing-related requests